src/main/java/org/jnode/fs/xfs/inode/INode.java

```diff
@@ -80,6 +150,24 @@ public int getMode() {
         return getUInt16(0x2);
     }
 
+    /**
+     * Is Directory the current inode.
+     *
+     * @return true.
+     */
+    public boolean isDirectory() throws IOException {
+        return FileMode.is((int) getMode(),FileMode.DIRECTORY);
+    }
```

Comment:
```suggestion
     * @return {@code true} if this is a directory, {@code false} otherwise.
     */
    public boolean isDirectory() {
        return FileMode.is(getMode(), FileMode.DIRECTORY);
    }
```

Reply:
The suggestion was applied, thanks Erwin.